vendor/golang.org/x/sys/unix/ztypes_freebsd_386.go

```diff
@@ -362,7 +362,7 @@ type FpExtendedPrecision struct{}
 type PtraceIoDesc struct {
 	Op   int32
 	Offs uintptr
-	Addr uintptr
+	Addr *byte
 	Len  uint32
 }
 
```

Comment:
It looks like a small change where the `Addr` field in the `PtraceIoDesc` struct was changed from a `uintptr` type to a pointer to a byte (`*byte`). This could be an intentional change to correspond to other parts of the code or to make the code more readable. However, without more context, it is difficult to say for certain if this change will introduce any bug risk or if there are any improvement suggestions that would be particularly useful.